doc/articles/getting-started/counterapp/includes/include-mvux.md

```diff
@@ -13,35 +13,62 @@ As part of creating the application, we selected MVUX as the presentation framew
     }
     ```
 
-- Add the `Count` and `Step` properties, to the `MainModel` class. These fields both must be of type `IState<int>` and to initialize them we use `=> State.Value(...)`.
+- Add a new `partial record` above named `Countable`. This record will be responsible for updating our counter's properties all while ensuring immutability. Learn more about immutable records [here](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Mvux/WorkingWithRecords.html#how-to-create-immutable-records).
 
     ```csharp
-    public IState<int> Count 
-        => State.Value(this, () => 0);
+    internal partial record Countable
+    {
+    }
+    ```
 
-    public IState<int> Step 
-        => State.Value(this, () => 1);
+- Add the `Count` and `Step` properties to the `Countable`'s primary constructor.
+
+    ```csharp
+    internal partial record Countable(int Count, int Step)
+    {
+    }
     ```
 
-- Add a method called `IncrementCommand` to the `MainModel` that will increment the counter by the step size. The generated ViewModel of our `MainModel` will automatically re-expose as `ICommand` the `IncrementCommand` method.
+- Add an `Increment` method to the `Countable` record. The `with` operator allows us to [create a new instance](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Mvux/WorkingWithRecords.html#updating-records) of the object.
```

Comment:
```suggestion
- Add an `Increment` method to the `Countable` record. The `with` operator allows us to [create a new instance](xref:Uno.Extensions.Mvux.Records#updating-records) of the object.
```

Reply:
@rajamatt Make sure to verify that the link is properly working while running the doc locally please